tfhe/docs/how_to/debug.md

```diff
@@ -0,0 +1,66 @@
+# Debugging FHE code
+
+Since tfhe-rs 0.5, [trivial ciphertexts](./trivial_ciphertext.md) have another application.
+They can be used to allow debugging via a debugger or print statements as well as speeding-up execution time
+so that you won't have to spend minutes waiting for execution to progress.
+
+This can greatly improve the pace at which one develops FHE applications.
+
```

Comment:
There needs to be a big fat warning about trivial encryptions not being secure I would think

Reply:
added just below